tests/admin_filters/tests.py

```diff
@@ -1150,7 +1172,7 @@ def test_simplelistfilter(self):
 
         # Make sure the correct queryset is returned
         queryset = changelist.get_queryset(request)
-        self.assertEqual(list(queryset), list(Book.objects.order_by("-id")))
+        self.assertSequenceEqual(list(queryset), list(Book.objects.order_by("-id")))
```

Comment:
`list()` is unnecessary when you use `assertSequenceEqual()`:
```suggestion
        self.assertSequenceEqual(queryset, Book.objects.order_by("-id"))
```

Reply:
`self.assertSequenceEqual(queryset, list(Book.objects.order_by("-id")))`
I can remove the `list()` from queryset, but I need it wrapping the other one. Not sure why though, to have the query evaluated maybe? :thinking:

Reply:
Ah yes because we have two querysets :+1: